content/docs/reference/branding/error-message-first-paragraph.md

```diff
@@ -0,0 +1,6 @@
+# Error Messages Header
+- Type: `string` containing [markdown](https://www.markdownguide.org/basic-syntax/
+- Example: `Please see https://your.internal.supportsite.com/how-do-remediate-something for details.`
+- Default: ''
+
+If set, will add the paragraph or markdown to every Error Details page. Used when Pomerium throws a 403 Unauthorized.
```

Comment:
```suggestion
If an error message is set, this will add the paragraph or markdown to every Error Details page. It is used when Pomerium throws a 403 Unauthorized.
```